.github/workflows/commit-checks.yml

```diff
@@ -0,0 +1,24 @@
+name: commit checks
+on:
+  push:
+jobs:
+  run:
+    name: test
```

Comment:
This yaml has 2 names with the word `test`. Maybe you can make it a little bit more explicit what is tested